test/Debugger/JsDiagGetStackProperties.js.dbg.baseline

```diff
@@ -0,0 +1,107 @@
+[
+  {
+    "this": "Object {...}",
+    "{exception}": "number 1",
+    "locals": {
+      "FuncLevel1": "function <large string>",
+      "outerFunc1": "function <large string>",
+      "globalVar": "undefined undefined"
+    }
+  },
+  {
+    "this": "Object {...}",
+    "{exception}": "Error Caught Error",
+    "locals": {
+      "FuncLevel1": "function <large string>",
+      "outerFunc1": "function <large string>",
+      "globalVar": "undefined undefined"
+    }
+  },
+  {
+    "this": {
+      "FuncLevel1": "function <large string>",
+      "outerFunc1": "function <large string>",
+      "globalVar": "Object {...}"
+    },
+    "arguments": {
+      "#__proto__": "Object {...}",

```

Comment:
if we add __proto__ as property it changes the internal prototype and we can't traverse the JSON recursively in dbgcontroller.js so I added #__proto__

PDM baseline will not match to JSRT dbg baseline, I initially tried but that's too much work and prevents us correctly verifying JSRT baseline. I plan to have same test file (.js) but different baseline for PDM and JsRT debugger tests.